### PR TITLE
Quickfix - add 'Fortbildungen' to topbar

### PR DESCRIPTION
**Pull request by Ojself**
https://ticketsystem.schul-cloud.org/browse/SC-4979
Added one more navigation item. See more information in ticket.
# Screenshots

nuxt
legacy

### CODE REVIEW

**Comment by BlackGhostHD** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
please add screenshot to your PR, as asked in the template. thanks

File: src/components/base/BaseIcon.vue [hunk 1]
```diff
@@ -90,6 +90,14 @@ export default {
...
 }
 
+.custom {
+	width: 1em;
+	/* stylelint-disable-next-line sh-waqar/declaration-use-variable */
+	font-size: calc(1em + 4px);
+	line-height: 100%;
+	vertical-align: middle; // should this be default?
+}
+
```
Comment (BlackGhostHD) [inline src/components/base/BaseIcon.vue:100]: does this have a visible effect on already existing or used icons?

**Comment by Ojself** [comment]
nuxt
legacy

File: src/components/base/BaseIcon.vue (hunk 1 above)
Comment (Ojself) [reply src/components/base/BaseIcon.vue:100]: No

**Comment by Ojself** [comment]
@schul-cloud-bot update screenshots

**Comment by Ojself** [comment]
@schul-cloud-bot update screenshots

**Comment by Ojself** [comment]
@dominikbruechner 
The screenshots were in the ticket, but are now posted in this pull request.

**Pull request merged**